src/kep/vintage.py

```diff
@@ -42,12 +46,14 @@ class Latest(Vintage):
 
     def __init__(self, year: int, month: int):
         super().__init__(year, month)
-        # protect from using old realeses of data 
+        # protect from using old releases of data
         Date(year, month).assert_latest()
 
-    def upload(self, password: str):        
```

Comment:
I didn't understand why you need `password` arg since you're using API key in `Uploader`.